src/System Application/App/Permission Sets/src/PermissionSetRelationImpl.Codeunit.al

```diff
@@ -398,7 +448,22 @@ codeunit 9856 "Permission Set Relation Impl."
         exit(10);
     end;
 
-    procedure LookupPermissionSet(AllowMultiselect: Boolean; var AggregatePermissionSet: Record "Aggregate Permission Set"): Boolean
+    procedure LookupAllPermissionSets(AllowMultiselect: Boolean; var PermissionSetBuffer: Record "PermissionSet Buffer"): Boolean
+    var
+        PermissionSetLookupList: Page "Permission Set Lookup List";
+    begin
+        PermissionSetLookupList.LookupMode(true);
+        if PermissionSetLookupList.RunModal() = Action::LookupOK then begin
```

Comment:
It seems a bit odd to me that the page itself doesn't know if multi-select is allowed. That sounds like something it should know and tell the user. Otherwise the user may try to multi-select when it's not allowed and vice versa. Would be nice info to the user :)

Reply:
I'm not sure how to communicate that to the user. Multi select is allowed when using the "Select permission sets" action and it's not allowed when adding a single permission set through the drilldown - a similar pattern is applied on other pages in the application